droidlet/dashboard/web/src/StateManager.js

```diff
@@ -214,8 +214,10 @@ class StateManager {
       this.memory = this.initialMemoryState;
       // clear state of all components
       this.refs.forEach((ref) => {
-        ref.setState(ref.initialState);
-        ref.forceUpdate();
+        if (!(ref instanceof TimelineDetails)) {
```

Comment:
This is to keep detail panes open when a disconnect event occurs right?

Reply:
This is to preserve the contents of each detail pane when a disconnect event occurs. Otherwise, forceUpdate() causes each panel to rerender and show the same (latest) click event information (i.e. we'll have multiple tabs displaying the same information regardless of what was clicked)